### ISSUE TITLE
1.71 RIFE Vulkan doesn't work

### ISSUE THREAD

**Issue by applesfruit** [f0e/blur#8]
Pretty sure RIFE for Vulkan is rife-ncnn right? I inputted that and I get an error? Does it use a different parameter or?

**Comment by couleurm** [comment]
> Pretty sure RIFE for Vulkan is rife-ncnn right? I inputted that and I get an error? Does it use a different parameter or?

Installer seems broken, I haven't found a manual way to install it yet, let me know

**Comment by applesfruit** [comment]
> > Pretty sure RIFE for Vulkan is rife-ncnn right? I inputted that and I get an error? Does it use a different parameter or?
> 
> Installer seems broken, I haven't found a manual way to install it yet, let me know

Yeah uh I found a github repo of rife-ncnn-vulkan. Is there anyway using this to make rife-ncnn work?
Github Repo: https://github.com/nihui/rife-ncnn-vulkan

**Comment by applesfruit** [comment]
> > > Pretty sure RIFE for Vulkan is rife-ncnn right? I inputted that and I get an error? Does it use a different parameter or?
> > 
> > 
> > Installer seems broken, I haven't found a manual way to install it yet, let me know
> 
> Yeah uh I found a github repo of rife-ncnn-vulkan. Is there anyway using this to make rife-ncnn work? Github Repo: https://github.com/nihui/rife-ncnn-vulkan

The parameter "vsrife" for the interpolation program works. It probably stands for VapourSynth RIFE, as that is what is used to interpolate the video (VSPipe.exe)

**Comment by couleurm** [comment]
oh you made it work? what did you install

**Comment by applesfruit** [comment]
> oh you made it work? what did you install

Nothing. Just input "vsrife" in the interpolation program parameter.

**Comment by f0e** [comment]
Bulk-closing old issues due to 2.0 release. RIFE isn't currently supported but I'm looking to reimplement it. For now, use [smoothie-rs](https://github.com/couleur-tweak-tips/smoothie-rs) if you need RIFE.
